### PR TITLE
🎁: Implemented Report access information

### PR DESCRIPTION
**Pull request by kjellhaaland**
Updated PowerBI Component to support showing more information related to report access when the user does not have access to the report.

- Added avatar for the person responsible for the report.
- Added report general description
- Added report access description.
- Added new prop for PBI reports, that provides the dialog with information.

Implements: equinor/fusion-workspace#551

### CODE REVIEW

File: packages/power-bi/src/lib/components/error/ReportErrorMessage.tsx [hunk 1]
```diff
@@ -1,62 +1,108 @@
...
+            <StyledReportDetailsWrapper>
+              <StyledReportDescriptionWrapper>
+                <Typography variant="h5">Report description</Typography>
+                <Markdown>{accessInfo.description}</Markdown>
+              </StyledReportDescriptionWrapper>
+
+              <StyledReportOwnerWrapper>
+                <Typography variant="h5">Report owner</Typography>
+                <Card.Header style={{ paddingTop: '20px', paddingBottom: '20px' }}>
+                  <PersonAvatar azureId={accessInfo.ownedBy.azureUniqueId} />
```
Comment (Gustav-Eikaas) [inline packages/power-bi/src/lib/components/error/ReportErrorMessage.tsx:84]: This is a tricky one. Albeit the name is very confusing (workspace-**fusion**). We are trying to separate this whole repository from Fusion and letting the consumers (cc-comps) implement/provide the fusion specifics. This component will have to make an api call with an authenticated client to get the person avatar. What we have done in scenarios like this previously is to add a prop to the component that is something like "ErrorComponent" and implementing all the logic in CC components/shared

File: packages/power-bi/src/lib/components/PowerBi.tsx [hunk 2]
```diff
@@ -17,6 +17,7 @@ export interface PowerBiProps {
   getToken: (reportUri: string, signal?: AbortSignal) => Promise<FusionPowerBiToken>;
   getEmbedInfo: (reportUri: string, token: string, signal?: AbortSignal) => Promise<FusionEmbedConfig>;
   getErrorMessage: (reportUri: string) => Promise<string>;
+  getAccessInfo: (reportUri: string) => Promise<ReportAccessInfo>;
```
[the remark this answers is not in the document]
Comment (kjellhaaland) [reply packages/power-bi/src/lib/components/PowerBi.tsx]: Removed these functions, and added support for an Error component.

**Comment by kjellhaaland** [comment]
@Gustav-Eikaas, this fix now introduces a breaking change.
Should i follow semver and bump to the following?

- @equinor/workspace-powerbi from 1.0.16 to 2.0.0
- @equinor/workspace-fusion from 7.0.6 to 8.0.0
-

**Review by Gustav-Eikaas — APPROVED** [review]
LGTM! Great work!
Bump majors yes

**Pull request merged**